requirements/base.txt

```diff
@@ -0,0 +1 @@
+django-polaris>=1.4.1
```

Comment:
I think you'll need to use Polaris' `custodial-support` branch, not a published version.

Also, if you do use a published version use v2.0+

Reply:
Yes, I only added it to the requirements for some developing purpose (like autocomplete for other Polaris import and Stellar SDK installation). For deployment, we're going to use Polaris locally and on that branch.

In the future, when Polaris' `custodial-support` branch beings published, we'll modify this requirement.